MPChartLib/src/main/java/com/github/mikephil/charting/components/YAxis.java

```diff
@@ -38,6 +38,27 @@
      */
     protected boolean mDrawZeroLine = false;
 
+    /**
+     * flag indicating that auto scale min restriction should be used
+     */
+
+    private boolean mUseAutoScaleRestrictionMin = false;
```

Comment:
set this to a single "use auto scale restriction" or something similar boolean flag.